controls.py

```diff
@@ -155,35 +112,6 @@ def request_info_who(post_n,type_n):
     return data
 # print(request_info()[1][0])

```

Comment:
maybe I am missing it but do we not have request_info() anymore? If not, how are we referencing db now?